ini/stm32f4.ini

```diff
@@ -23,78 +23,62 @@
 # ARMED (STM32)
 #
 [env:ARMED]
-platform      = ${common_stm32.platform}
```

Comment:
The `platform` field ***must*** be included or the PlatformIO IDE will not show it in its interface.

Reply:
Ah, lame. I was hoping it would just reference the parent environment. Good to know.

Reply:
I would prefer the IDE just showed all environments that start with `env:` but the IDE disagrees.

Reply:
This seems to be fixed in the latest VSCode/PIO. I just went through and commented out the the `platform =` lines and they still show up in the Project Tasks list.